### ISSUE TITLE
_Solve_ Mate in 1 puzzles

### ISSUE THREAD

**Issue by sriram-an** [sundarsrd/Learnings#1]

**Comment by sriram-an** [comment]
[5334 Chess Puzzles.pdf](https://github.com/sundarsrd/Learnings/files/5146390/5334.Chess.Puzzles.pdf)

**Comment by adityaparameswar** [comment]
uncle, you have created an issue , who is it for?
 
is it for me??

**Comment by sriram-an** [comment]
Hello Aditya,

You can start on it if you want to. Have a daily target of at least 50 puzzles per day. If you have any queries you can mail me. We can schedule the other topics under Chess sometime this week.

--
regards,
Sriram



On 31-Aug-2020, at 21:59, adityaparameswar <notifications@github.com<mailto:notifications@github.com>> wrote:



uncle, you have created an issue , who is it for?

is it for me??

**Comment by adityaparameswar** [comment]
yes uncle will do, and are we going to have daily online classes for coding???

because i have a lot of time at home and it would be easier for me to solve the puzzles if you explain me everyday.

**Comment by adityaparameswar** [comment]
good evening uncle, i did not start yet. 

as i did not know what to do, do i check mate the king ??
else what should i do??

**Comment by sriram-an** [comment]
Hi Aditya,

Tomorrow we can have a chess session - let me know at around this time.

We can plan coding session from next week.


On 01-Sep-2020, at 17:56, adityaparameswar <notifications@github.com<mailto:notifications@github.com>> wrote:



good evening uncle, i did not start yet.

as i did not know what to do, do i check mate the king ??
else what should i do??

**Comment by adityaparameswar** [comment]
@sriram-an uncle: 
   =   Mate_in_1 solutions. 
1.move the queen f6-g7. 
2.move the queen h6-g6. 
3. move the queen h7-g7.
4.move the rook b3-h3.
5.move the rook from h1-h8.
6.move the rook from d1-g1.
7.move the rook from d1 -h1.
8.move the rook from d1-h1.
9.move the rook b1-g1.
10.move the bishop f1-f6.
11.move the rook d1-e1.
12.move the bishop d3-d4.
13.move the bishop from c4-f7.
14.move the queen from d1-h5.
15.move the rook from e1-e8.
16.move the queen from h5-h7.
17.move the rook from d1-d8.
18.move the knight from e5-g6.
